tests/settings.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'vat',
+        'user': '',
+        'password': '',
+    }
+}
```

Comment:
Can we parse DATABASES settings from `DATABASE_URL` variable? It will fix problems with running the tests in various CI environments.